util/test/checkChplInstall

```diff
@@ -189,40 +200,63 @@ else
     log_warning "See \$CHPL_HOME/doc/launcher.rst for information on manually launching Chapel programs."
     # This exit code should be recognized as a failure to complete check, but
     # not necessarily failure of build
+    log_debug "Exit \"make check\" script with status code 10"
     exit 10
 fi
 
 TEST_EXEC_OUT=${TEST_JOB}.exec.out
 
 # Run compiled binary with parsed execution options
 (
-    cd ${TMP_TEST_DIR}
+    cd ${TMP_TEST_DIR} || { log_error "failed cd $TMP_TEST_DIR" ; exit 1 ; }
 
     EXEC_OPTS="$(cat ${TEST_DIR}/${TEST_JOB}.execopts)"
 
     log_info "Running test job."
-    ./${TEST_JOB} -nl${NUMLOCALES} ${EXEC_OPTS} | sort > ${TEST_EXEC_OUT} 2>&1
+    ./${TEST_JOB} -nl${NUMLOCALES} ${EXEC_OPTS} | sort > ${TEST_EXEC_OUT}
```

Comment:
Yes. Even if not, the only thing on stderr right there would be errors from "sort" itself. 
A better question is, why isn't the stderr from running the test job redirected into THAT stdout stream? It probably should be. But, that would NOT be consistent with sub_test. And, I'm not about to change sub_test.